### ISSUE TITLE
Outdated documention for Life()

### ISSUE THREAD

**Issue by mihalybaci** [cesaraustralia/DynamicGrids.jl#67]
I was playing around with the "game of life" simulator ran into some trouble following the documentation. The help for `Life` yields
```
# Life. 
  using DynamicGrids, Distributions
  init = Bool.(rand(Binomial(1, 0.5), 70, 70))
  output = REPLOutput(init; tspan=(1, 1000), fps=10, color=:red)
```
However, the command for `REPLOutput` (as well as `ArrayOutput`) yield an error when `tspan` is specified as a tuple. I presume this is just a documentation error since the output types both give `tspan::AbstractRange`.

**Comment by rafaqz** [comment]
Yep it's just documentation. It' used to accept a tuple - use a range now.

**Comment by rafaqz** [comment]
That code is actually run by documenter - it shouldn't be allowed to fail. 

I'll fix this it shouldn't happen.

**Comment by rafaqz** [comment]
This is fixed for 0.10.8. Do try the examples again they actually look really cool in the terminal :)

**Comment by mihalybaci** [comment]
Great! Thank you very much!
